config/locales/en.yml

```diff
@@ -574,7 +574,12 @@ en:
   depth: "Depth"
   payment_could_not_process: "The payment could not be processed"
   payment_could_not_complete: "The payment could not be completed"
-
+  vine_voucher_validator_service:
+    errors:
+      vine_settings: "No Vine api settings for the given enterprise" 
+      vine_api: "There was an error communicating with the API"
+      invalid_voucher: "The voucher is not valid"
+      not_found_voucher: "The voucher doesn't exist"
```

Comment:
These may be updated by the instance manager to be more contextual, but I feel we can still start with more helpful messages from the customer's perspective. I'm not sure how we are communicating the type of voucher, but assuming the user would know what "VINE" means, how about:
```
      vine_settings: "This shop is not enabled for VINE Vouchers." 
      vine_api: "There was an error communicating with the API, please try again later."
      invalid_voucher: "The voucher is not valid"
      not_found_voucher: "Sorry, we couldn't find that voucher, please check the code."
```

For `invalid_voucher`, I'm not sure what might cause an invalid voucher, or who to contact if that's the case. Eg if the balance has already been used, how will the user be able to find out? Maybe we can pass through the error message from the API.

Reply:
Not all these error are forwarded to the customer, but I like your suggestion.
re Invalid voucher, I am waiting on Rachel's feedback : https://github.com/openfoodfoundation/openfoodnetwork/issues/12859#issuecomment-2443257975